src/xdist/remote.py

```diff
@@ -24,6 +24,24 @@ def setproctitle(title):
         pass
 
 
+class Producer:
+    """this is defined here because this module can't depend on xdist"""
```

Comment:
```suggestion
    """
    Simplified implementation of the same interface as py.log, for backward compatibility
    since we dropped the dependency on pylib.
    Note: this is defined here because this module can't depend on xdist, so we need 
    to have the other way around.    
    """
```

Reply:
Will go ahead and apply this suggestion as I want to move #821 forward. 👍